CC/include/CCGeom.h

```diff
@@ -227,6 +227,7 @@ template <typename Type> class Vector3Tpl : public Tuple3Tpl<Type>
 	static inline void vset(Type p[], Type s) {p[0]=p[1]=p[2]=s;}
 	static inline void vset(Type p[], Type x, Type y, Type z) {p[0]=x; p[1]=y; p[2]=z;}
 	static inline void vadd(const Type p[], const Type q[], Type r[]) {r[0]=p[0]+q[0]; r[1]=p[1]+q[1]; r[2]=p[2]+q[2];}
+	// note misspelling: should be vsubtract
 	static inline void vsubstract(const Type p[], const Type q[], Type r[]) {r[0]=p[0]-q[0]; r[1]=p[1]-q[1]; r[2]=p[2]-q[2];}
```

Comment:
Note that this method is misspelled "vsubstract", should be "vsubtract", but I didn't want to touch source code that may affect 3rd parties.